tbx/project_styleguide/templates/patterns/molecules/streamfield/streamfield-example-standard.yaml

```diff
@@ -0,0 +1,40 @@
+context:
+  blockquote_client_data:
+    quote: Torchbox's collaboration has been delightful, significantly elevating NHS.UK team's appreciation of SEO. They're diligent, eager, and proactive, consistently delivering optimal solutions for any challenge
+    company_logo: True
+    name: Theo Smart
+    role: Head of Digital, NHS UK
+    link:
+      0:
+        value:
+          url: '#'
+          text: Book a free 1-hour call with Michael
+
+tags:
+  image:
+    avatar width-143 format-webp class="avatar__image":
```

Comment:
the avatar image can go here as it isn't used by the default pullquote (also it will save updating it when my changes in https://github.com/torchbox/torchbox.com/pull/32 are merged)

Reply:
Good spot! Removed in https://github.com/torchbox/torchbox.com/pull/33/commits/7285261a3e2ddc9e5209b15f5e86e681cfba049e